org.learningequality.Kolibri.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.learningequality.Kolibri",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--share=network",
+    "--filesystem=~/.kolibri:create"
+  ],
+  "command": "open_kolibri.sh",
+  "modules": [
+    {
+      "name": "kolibri",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install --prefix=/app .",
+        "sed -i '/<\\/head>/i\\<script>var newImage = new Image(); setInterval(function() {newImage.src = \"http://127.0.0.1:\" + (parseInt(location.port) + 1) + \"/heartbeat.png?\" + new Date().getTime();}, 30000);</script>' /app/lib/python3.7/site-packages/kolibri/core/templates/kolibri/base.html"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/3d/f1/950058edaf24009d35729ed88cc5da480e34f911cdc3a5a0f7de49ce1707/kolibri-0.12.8.tar.gz",
+          "sha256": "a60488472c85a83baf3faa1fd94b47a86486830069c0b7fc3acb376234acf7f6"
+        }
+      ]
+    },
+    {
+      "name": "helper_scripts",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D envvars.sh /app/bin/envvars.sh",
+        "install -D run_kolibri.sh /app/bin/run_kolibri.sh",
+        "install -D open_kolibri.sh /app/bin/open_kolibri.sh",
+        "install -D check_for_running_tasks.sh /app/bin/check_for_running_tasks.sh",
+        "install -D redirect_server.py /app/www/redirect_server.py",
+        "install -D index.htm /app/www/index.htm",
+        "install -D spinner.gif /app/www/spinner.gif",
+        "install -D heartbeat.png /app/www/heartbeat.png"
+      ],
+      "sources": [
+        {
+          "type": "dir",
+          "path": "src/"
+        }
+      ]
+    },
+    {
+      "name": "desktop",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm644 org.learningequality.Kolibri.appdata.xml /app/share/appdata/org.learningequality.Kolibri.appdata.xml",
```

Comment:
Ok, thanks for noting. We don't currently have any OS-specific pieces in the upstream repository, as it is a cross-platform project. We generally maintain the OS-specific installers/launchers/etc as separate repositories (e.g. for [Windows](https://github.com/learningequality/kolibri-installer-windows) and [Debian](https://github.com/learningequality/kolibri-installer-debian)). If it would be preferred, we could set up a separate upstream repository for these files rather than having them in the Flathub repo?

Reply:
Overall that would be best but it's not a blocker. We're just trying to encourage a) making things better for everybody and b) making sure that the flatpak users use is as close to the system that developers develop on as possible. Fewer bugs and easier fixes that way!